src/blocks/battery.rs

```diff
@@ -508,7 +508,12 @@ impl Block for Battery {
             self.output.set_icon(match status.as_str() {
                 "Discharging" => "bat_discharging",
                 "Charging" => "bat_charging",
-                _ => "bat",
+                _ => match capacity {
+                    Ok(0...15) => "bat_quarter",
```

Comment:
May I ask why you chose these particular ranges?

Reply:
Originally, I wanted to match the ranges of the coloring and after that I realized that there are only icons for a quarter :sweat_smile: 

But I guess that it's more intuitive to use quarters for the range :+1:

Reply:
fixed :)